src/client/components/Login.tsx

```diff
@@ -0,0 +1,132 @@
+import React, { Fragment, useState, useRef } from "react";
+import ReactDOM from "react-dom/client";
+import "../styles/Login.css";
+
+import { BrowserRouter as Router, Route, Link } from "react-router-dom";
+import Owner from "./Owner";
+
+interface ErrorMessage {
+  name: string;
+  message: string;
+}
+
+interface FormValue {
+  name: string;
+  pass: string;
+}
+
+function Login() {
+  //seed data
+  const database = [
+    {
+      username: "user1",
+      password: "pass1",
+    },
+    {
+      username: "user2",
+      password: "pass2",
+    },
+  ];
+
+  // React States
+  const [errorMessages, setErrorMessages] = useState<ErrorMessage>({
+    name: "",
+    message: "",
+  });
+  const [isSubmitted, setIsSubmitted] = useState<boolean>(false);
```

Comment:
I think the variable name should be fixed.
Because this variable usage is when user signed in successfully, not when user clicks the submit button.

Reply:
Yes